package.json

```diff
@@ -47,6 +47,7 @@
     "@babel/core": "7.7.7",
     "@babel/plugin-proposal-class-properties": "7.7.4",
     "@babel/plugin-proposal-object-rest-spread": "7.7.7",
+    "@babel/plugin-proposal-optional-chaining": "7.8.3",
```

Comment:
let's get this in a pre-PR

Reply:
It was added by default to preset env: https://github.com/babel/babel/releases/tag/v7.8.0

We just need to rebase in [this PR](https://github.com/Automattic/simplenote-electron/pull/1848) after it is merged